plugin/sim/simarray.py

```diff
@@ -232,7 +232,7 @@ def __init__(self, statefile, timeout):
             rpm INTEGER,
             link_type INTEGER,
             FOREIGN KEY(owner_pool_id)
-            REFERENCES pools(id) ON DELETE CASCADE );
+            REFERENCES pools(id) ON DELETE SET DEFAULT);
```

Comment:
Here, should be `ON DELETE SET NULL`? As, owner_pool_id seems doesn't have default value.